### PR TITLE
29 July Update

### PR DESCRIPTION
**Pull request by gabriel-seow**
- RW
- CG
- PG
- VN
- BE
- LB
- BG
- HN
- KE
- HK
- GH
- MG
- UZ
- JM
- TN
- ZA

### CODE REVIEW

File: src/constants/travel.yaml [hunk 1]
```diff
@@ -334,6 +336,7 @@ countries:
         - '[A 50% occupancy condition applies](http://www.mh.government.bg/bg/novini/aktualno/ot-utre-23-yuni-noseneto-na-predpazna-maska-ili-dr/) to the holding of cultural and entertainment events indoors and outdoors.'
       inbound:
         - 'All nationals, permanent residents and their family members of the European Union, United Kingdom, the Schengen Agreement States including San Marino, Andorra, Monaco and Vatican City, Bosnia and Herzegovina, Montenegro, and Serbia, are [allowed into Bulgaria as of June 18, 2020](https://bg.usembassy.gov/novel-coronavirus-covid-19-information/#:~:text=Entry%20ban%3A%20U.S.%20citizens%20are,exceptions%20to%20this%20entry%20ban.&text=As%20of%20June%2018%2C%20permanent,countries%20may%20enter%20%E2%80%8BBulgaria).'
+        - 'Tourists from Serbia, North Macedonia, Albania, Kosovo, Bosnia, Montenegro, Moldova, Israel, and Kuwait will be allowed to visit the country however, they must provide proof of a negative COVID-19 test, [taken no more than 72 hours before arrival](https://www.garda.com/crisis24/news-alerts/363836/bulgaria-travel-bans-to-ease-for-tourists-from-select-countries-from-july-28-update-14).'
```
Comment (jannwong) [inline src/constants/travel.yaml:339]: full stop before `however`. and capitalize `H`

File: src/constants/travel.yaml [hunk 2]
```diff
@@ -334,6 +336,7 @@ countries:
...
       visa_quarantine:
         - 'All persons (with limited exceptions) arriving in Bulgaria from Sweden, the United Kingdom, Portugal, and from all third countries (outside the European Union), except Serbia, Bosnia and Herzegovina and Montenegro, Iceland, Norway, Switzerland, Lichtenstein, San Marino, Andorra, Monaco, and Vatican City must [quarantine for a period of 14 days at home or at other accommodation](https://bg.usembassy.gov/novel-coronavirus-covid-19-information/#:~:text=Entry%20ban%3A%20U.S.%20citizens%20are,exceptions%20to%20this%20entry%20ban.&text=As%20of%20June%2018%2C%20permanent,countries%20may%20enter%20%E2%80%8BBulgaria).'
```
Comment (jannwong) [inline src/constants/travel.yaml:341]: this sentence is very confusing.
All persons (with limited exceptions) arriving in Bulgaria from Sweden, the United Kingdom, Portugal, and from all countries outside the European Union must [quarantine for a period of 14 days at home or at other accommodation](https://bg.usembassy.gov/novel-coronavirus-covid-19-information/#:~:text=Entry%20ban%3A%20U.S.%20citizens%20are,exceptions%20to%20this%20entry%20ban.&text=As%20of%20June%2018%2C%20permanent,countries%20may%20enter%20%E2%80%8BBulgaria).' This excludes visitors from Serbia, Bosnia and Herzegovina and Montenegro, Iceland, Norway, Switzerland, Lichtenstein, San Marino, Andorra, Monaco, and Vatican City.

File: src/constants/travel.yaml [hunk 3]
```diff
@@ -607,6 +620,8 @@ countries:
       domestic:
         - 'The Republic of Congo has closed its land, air, and maritime borders due to the ongoing pandemic and [domestic and international commercial flights currently remain suspended](https://www.garda.com/crisis24/news-alerts/353951/congo-number-of-confirmed-covid-19-cases-passes-1000-on-june-23-as-state-of-emergency-extended-update-7).'
         - 'Non-essential businesses, schools, and places of worship also [remain closed](https://www.garda.com/crisis24/news-alerts/353951/congo-number-of-confirmed-covid-19-cases-passes-1000-on-june-23-as-state-of-emergency-extended-update-7).'
+        - 'Curfew hours in Brazzaville and Pointe-Noire will be extended, between 8 pm and 5 am (local time) from July 28, 2020](https://www.garda.com/crisis24/news-alerts/363941/congo-authorities-extend-state-of-health-emergency-until-august-19-update-9).'
```
Comment (jannwong) [inline src/constants/travel.yaml:623]: missing `[`

File: src/constants/travel.yaml [hunk 4]
```diff
@@ -607,6 +620,8 @@ countries:
...
+        - 'Domestic passenger flights and road travel [have resumed nationwide](https://www.garda.com/crisis24/news-alerts/363941/congo-authorities-extend-state-of-health-emergency-until-august-19-update-9).'
```
Comment (jannwong) [inline src/constants/travel.yaml:624]: missing `[`

File: src/constants/travel.yaml [hunk 5]
```diff
@@ -2014,12 +2030,18 @@ countries:
       outbound: partial
   JM:
     sources:
+      partial:
+        - 'Movement restrictions apply and tourists may only travel within the designated [Resilient Corridor.](https://www.jtbonline.org/wp-content/uploads/FAQs-for-Re-Opening-July-10.pdf).'
       inbound:
         - 'All persons travelling to Jamaica require a [Travel Authorization](https://www.visitjamaica.com/travelauthorization/).'
```
Comment (jannwong) [inline src/constants/travel.yaml:2036]: single `l` for travelling

File: src/constants/travel.yaml [hunk 6]
```diff
@@ -2215,14 +2237,18 @@ countries:
     sources:
       domestic:
         - 'Kenyatta stated that international commercial flights would resume from August 1, 2020, and that [domestic flights have resumed since July 15, 2020](https://www.garda.com/crisis24/news-alerts/357071/kenya-movement-restrictions-to-be-lifted-in-nairobi-mombasa-and-mandera-counties-july-7-update-25).'
-        - 'The government extended the current curfew in place between 21:00 and 04:00 by another 30 days as of July 6, 2020. [Public gatherings also remain banned for a further 30 days](https://www.garda.com/crisis24/news-alerts/357071/kenya-movement-restrictions-to-be-lifted-in-nairobi-mombasa-and-mandera-counties-july-7-update-25).'
+        - 'The government extended the current curfew in place between 21:00 and 04:00 by another 30 days as of July 27, 2020. Public gatherings also remain banned and [face masks must be worn in public](https://www.garda.com/crisis24/news-alerts/363811/kenya-nightly-curfew-extended-for-additional-30-days-july-27-update-27).'
```
Comment (jannwong) [inline src/constants/travel.yaml:2240]: the time format we have is 9 pm and 4 am

File: src/constants/travel.yaml [hunk 7]
```diff
@@ -2345,6 +2371,7 @@ countries:
     sources:
       domestic:
         - 'Public gatherings are to remain banned and cultural venues will not reopen and other various measures will be [extended until August 2, 2020](https://www.garda.com/crisis24/news-alerts/355786/lebanon-authorities-announce-extension-of-covid-19-restrictions-until-august-2-update-26).'
+        - 'Essential businesses, including grocery stores, gas stations, banks, and schools will be open on Tuesday and Wednesday only with a [near-total lockdown in place from Thursday until Monday.](https://www.garda.com/crisis24/news-alerts/363841/lebanon-lockdown-reimposed-amid-covid-19-spike-july-27-update-27)'
```
Comment (jannwong) [inline src/constants/travel.yaml:2374]: confusing sentence

Essential businesses, including grocery stores, gas stations, banks, and schools will be open on Tuesday and Wednesday; with a [near-total lockdown in place from Thursday until Monday.](https://www.garda.com/crisis24/news-alerts/363841/lebanon-lockdown-reimposed-amid-covid-19-spike-july-27-update-27)'

File: src/constants/travel.yaml [hunk 8]
```diff
@@ -2878,7 +2905,7 @@ countries:
   MG:
     sources:
       domestic:
-        - 'Authorities announced on Sunday, July 5, that full lockdown measures that had previously been imposed, [will be reintroduced](https://www.garda.com/crisis24/news-alerts/356801/madagascar-lockdown-measures-reimposed-in-analamanga-region-from-july-6-update-12) in the Analamanga region, including Antananarivo, from Monday, July 6, 2020, and will remain in place until July 25, 2020.'
+        - 'The full lockdown measures that had previously been imposed in the Analamanga region, including Antananarivo, from Monday, July 6, 2020, and will remain in place until August 10, 2020.](https://www.garda.com/crisis24/news-alerts/363616/madagascar-covid-19-state-of-emergency-extended-until-august-9-update-14)'
```
Comment (jannwong) [inline src/constants/travel.yaml:2908]: Monday is not required

File: src/constants/travel.yaml [hunk 9]
```diff
@@ -4046,21 +4077,26 @@ countries:
...
       visa_quarantine:
-        - 'Travelers arriving in Moscow from China, Iran, South Korea, Italy, France, Germany, Spain, United States and the United Kingdom are [required to self-quarantine for 14 days](https://www.garda.com/crisis24/news-alerts/347091/russia-over-400000-covid-19-cases-confirmed-in-country-as-of-may-31-update-39).'
+        - 'Foreign travelers arriving in Russia will be required to present negative COVID-19 test results that are [less than 72-hours old upon arrival](https://www.garda.com/crisis24/news-alerts/363596/russia-authorities-announce-resumption-of-flights-to-certain-destinations-from-august-1-update-43).'
     travel:
       domestic: partial
       inbound: 'no'
   RW:
+    reopening:
+      '2020-08-01': 'Kigali International Airport (KGL) will [reopen for commercial flights From August 1, 2020](https://www.garda.com/crisis24/news-alerts/356661/rwanda-authorities-to-reopen-airports-for-commercial-flights-from-august-1-update-10).'
```
Comment (jannwong) [inline src/constants/travel.yaml:4095]: flights `from` should not be capitalized

File: src/constants/travel.yaml [hunk 10]
```diff
@@ -4436,15 +4472,16 @@ countries:
...
+        - 'Domestic travel between cities and regions resumed on June 4, 2020, the [nationwide curfew was lifted on June 15, 2020](https://www.garda.com/crisis24/news-alerts/355241/tunisia-air-sea-and-land-borders-reopen-june-27-update-13).'
+        - 'Mosques, restaurants, and cafes [have reopened.](https://www.aa.com.tr/en/africa/tunisia-to-reopen-mosques-cafes-after-nearly-3-months/1849404)'
       inbound:
-        - '[On Saturday, June 27,  Tunisian authorities reopened the country''s air, sea, and land borders](https://www.garda.com/crisis24/news-alerts/355241/tunisia-air-sea-and-land-borders-reopen-june-27-update-13).'
-      outbound:
-        - '[On Saturday, June 27,  Tunisian authorities reopened the country''s air, sea, and land borders](https://www.garda.com/crisis24/news-alerts/355241/tunisia-air-sea-and-land-borders-reopen-june-27-update-13).'
+        - 'On June 27, 2020, Tunisian authorities [reopened the country''s air, sea, and land borders](https://www.garda.com/crisis24/news-alerts/355241/tunisia-air-sea-and-land-borders-reopen-june-27-update-13).'
```
Comment (jannwong) [inline src/constants/travel.yaml:4478]: put the date in reopen

File: src/constants/travel.yaml [hunk 11]
```diff
@@ -5014,6 +5052,8 @@ countries:
         - '[International travel is suspended indefinitely](https://www.garda.com/crisis24/news-alerts/354681/south-africa-government-to-ease-covid-19-restrictions-further-from-june-29-update-18).'
       outbound:
         - '[International travel is suspended indefinitely](https://www.garda.com/crisis24/news-alerts/354681/south-africa-government-to-ease-covid-19-restrictions-further-from-june-29-update-18).'
+      visa_quarantine:
+        - '[South Africa is currently testing an e-visa system for travelers who need to obtain a visa](https://www.southafricavisa.com/).'
```
Comment (jannwong) [inline src/constants/travel.yaml:5056]: hyperlink can be shortened

File: src/constants/travel.yaml (hunk 10 above)
Comment (gabriel-seow) [reply src/constants/travel.yaml:4478]: June 27 is already over!

Comment (jannwong) [reply src/constants/travel.yaml:4479]: we have a new feature coming up to state the date on the page.

**Pull request merged**